src/main/webapp/index.html

```diff
@@ -59,9 +59,8 @@ <h1 class="navbar-brand collapse-nav">Sections Manager <small id="serverInfo">{{
               <div class="form-group">
                 <label for="uniqname" class="sr-only">Enter uniqname</label>
                 <input type="text"  ng-model="uniqname" id="uniqname" class="form-control"  placeholder="uniqname" autofocus>
-
               </div>
-              <button id="uniqnameTrigger" type="button" ng-click="getCoursesForUniqname()" class="btn btn-primary">Look up courses <span ng-show="uniqname">for</span> {{uniqname}}</button>
+              <button id="uniqnameTrigger" type="button" ng-click="getCoursesForUniqname()" class="btn btn-primary">Look up courses <span ng-show="uniqname">for</span> {{uniqname}}</button> <span id="uniqnameValidMessage" class="alert alert-danger" style="display:none; padding:5px">Alpha chars only, please.</span>
```

Comment:
The checks for alphabetic characters only should resolve the problem of "@umich.edu" appearing with the uniqnames, as mentioned in TLUNIZIN-484.  However, shouldn't the error message also tell the user not to add that to the uniqname? Just telling them "alpha characters only" may not be understood by all users.